client/src/composables/fileDrop.ts

```diff
@@ -119,6 +129,7 @@ export function useFileDrop(
         "dragleave",
         () => {
             isFileOverDropZone.value = false;
+            setIframeEvents(iframesNoInteract, false);
```

Comment:
These watchers are for the center drop zone. Did you mean to include the events modifier here? It seems like it should be inside the finite state machine instead, especially since it's counterpart is also in the state machine.

Reply:
There really doesn't seem like a flexible enough place in the existing code for `setIframeEvents` to be called currently, so I can take a closer look.